server/Server.ts

```diff
@@ -205,16 +202,14 @@ class Server {
     this.app.set('views', [path.resolve(__dirname, 'static'), path.resolve(__dirname, 'templates')]);
     hbs.localsAsTemplateData(this.app);
     this.app.locals.config = {
-      APP_BASE: this.config.SERVER.APP_BASE,
-      BRAND_NAME: this.config.CLIENT.BRAND_NAME,
-      CHROME_ORIGIN_TRIAL_TOKEN: this.config.CLIENT.CHROME_ORIGIN_TRIAL_TOKEN,
```

Comment:
This was very old way to enable some custom features on Chrome :) 
Not used anymore :)